_i18n/fr.yml

```diff
@@ -155,853 +155,154 @@ global:
-      Les agences gouvernementales décident de façon indépendante comment interagir avec le public. Vous avez été redirigé(e) vers login.gov parce que le service ou le programme que vous souhaitez utiliser a choisi la plateforme partagée login.gov pour son efficacité et sa sécurité. Lorsque vous aurez terminé d'utiliser login.gov, vous serez redirigé(e) vers l'agence de départ.
-
-      Nous espérons que login.gov est une option pratique et sécuritaire pour vous. Cependant, si vous préférez ne pas utiliser login.gov, il demeure possible d'accéder aux services gouvernementaux. Vous trouverez plus d'information à ce sujet auprès de votre agence.
-    email-address-confirmation-link-is-invalid: |-
-      Le lien de confirmation envoyé par e-mail expire après 24 heures. Sélectionnez « envoyer de nouveau » pour obtenir un nouveau e-mail de confirmation ou pour recommencer le processus du début en sélectionnant « créer un compte » et en soumettant votre adresse e-mail.
+  get-started:
+    overview: |-
+      
+      You can add or delete email addresses from your login.gov Your Account page.
+
+      __To add an email address__ to your account, use the steps below:
+        1. Go to [https://secure.login.gov/](https://secure.login.gov/){:target="_blank"} and sign in to your account
+        2. Click on “Add email”
+        3. Enter the email address you want to add to your account and click Submit
+        4. You will receive an email in your inbox. Click on the link to confirm your email address.
+      The link will take you to the Your account page. You will see a green alert message confirming that your email address was added.
+
+      __To delete an email address__ from your account, use the steps below:
+      Note: you can’t delete an email address if it’s the only one linked to your account.
+        1. Go to [https://secure.login.gov/](https://secure.login.gov/){:target="_blank"} and sign in to your account
+        2. Click on the “Delete” link that is next to the email address you want to delete
+        3. Make sure the next page shows the correct email address that you want to delete
+        4. Click on “Delete email address”
+      You will see a green alert message confirming that your email address was removed.
+  trouble-signing-in:
+    overview: |-
+      # Des problèmes de connexion?
+      Vous avez oublié votre mot de passe? Verrouillé votre compte? Nous vous aiderons à résoudre les problèmes d’accès au compte.
+      
+      ## Sujets de dépannage courants
+      * [Comment se connecter à login.gov](site.baseurl/help/trouble-signing-in/how-to-sign-in)
+      * [Mot de passe oublié](site.baseurl/help/trouble-signing-in/forgot-your-password)
+      * [Verrouillé du compte login.gov](site.baseurl/help/trouble-signing-in)
+      * [Téléphone ou méthodes d’authentification non disponibles](site.baseurl/help/trouble-signing-in)
+      * [Adresse courriel oubliée](site.baseurl/help/trouble-signing-in)
```

Comment:
These links are 404. It's not clear to me if these are intended to be exempt by your note about "Soft launch will not include" items.

```suggestion
      * [Verrouillé du compte login.gov](site.baseurl/help/trouble-signing-in/locked-out-of-login)
      * [Téléphone ou méthodes d’authentification non disponibles](site.baseurl/help/trouble-signing-in/phone-or-authentication-methods-not-available)
      * [Adresse courriel oubliée](site.baseurl/help/trouble-signing-in/forgot-your-email-address)
```

To follow-up on: Why is our automated link testing not able to catch these?

Reply:
Good question. As I learned in this exercise, the translated urls are caught only if the English page is missing. If the English page is missing, the site build won't create the corresponding Spanish and French pages. Since, in this case, we have English content, the Spanish and French urls are valid.